cypress/e2e/debug_refresh.cy.js

```diff
@@ -10,6 +10,7 @@ describe('Refresh', () => {
     cy.get('[data-cy-debug="refresh"]').click();
     cy.wait(100);
     cy.get('[data-cy-debug="tableBody"]').find('tr').should('have.length', 1);
+    cy.get('[data-cy-debug="amountShown"]').should('have.text', '/1');
```

Comment:
Ik zou de '/' hier weglaten in de assertion, mochten we die weg willen laten in de toekomst dan blijft deze test nog werken namelijk.

Reply:
Verwerkt.